CMakeLists.txt

```diff
@@ -239,7 +239,7 @@ if (NOT USE_PRE_BUILT_NGRAPH)
     ExternalProject_Add(
         ext_ngraph
         GIT_REPOSITORY https://github.com/NervanaSystems/ngraph
-        GIT_TAG d3453447883fe48e63126316d5da72fcbd47dded
+        GIT_TAG df9e23761f5f7a7a7d0134c8ae0c92a8bad2e961
```

Comment:
This should be set to release tag i.e., `v0.15.0-rc.1` at this point. I will add this change.